### ISSUE TITLE
issue updating tabs dynamically?

### ISSUE THREAD

**Issue by chriddyp** [plotly/dash-core-components#265]
I haven't triaged this one, but this was reported by a community user: https://community.plot.ly/t/problem-creating-tabs-dynamically-using-callback/12728/4. We should make sure that the two methods that I listed do indeed work!

**Comment by valentijnnieman** [comment]
I've confirmed that:

- [x] Update the children property of some div with an entirely new dcc.Tabs instance
- [ ] Update the children property of a dcc.Tabs component with a new list of dcc.Tab instances.

works!

**Comment by radekwlsk** [comment]
For me when I generate `children` of a `html.Div`, that have multiple `dcc.Tabs`, dynamically with callback for each (dynamic tabs) and then try to change the `children` of that container `html.Div` it fails to replace it, throwing in front:
```
bundle.js?v=0.27.1:27 Uncaught (in promise) TypeError: Cannot read property 'props' of undefined
    at t.value (bundle.js?v=0.27.1:27)
    at p._renderValidatedComponentWithoutOwnerOrContext (react-dom@15.4.2.min.js?v=0.13.2:13)
    at p._renderValidatedComponent (react-dom@15.4.2.min.js?v=0.13.2:13)
    at p._updateRenderedComponent (react-dom@15.4.2.min.js?v=0.13.2:13)
    at p._performComponentUpdate (react-dom@15.4.2.min.js?v=0.13.2:13)
    at p.updateComponent (react-dom@15.4.2.min.js?v=0.13.2:13)
    at p.receiveComponent (react-dom@15.4.2.min.js?v=0.13.2:13)
    at Object.receiveComponent (react-dom@15.4.2.min.js?v=0.13.2:14)
    at p._updateRenderedComponent (react-dom@15.4.2.min.js?v=0.13.2:13)
    at p._performComponentUpdate (react-dom@15.4.2.min.js?v=0.13.2:13)
```

For me it looks like the callback for dynamic tab content of the already removed `dcc.Tabs` is called and it fails, as there is not `dcc.Tabs` to have `value` for that callback.

**Comment by radekwlsk** [comment]
@chriddyp doesn't #267 resolve the issues? What is stopping merge of those fixes? That issue really stops my Dash application development. Can I help in any way?

**Comment by chriddyp** [comment]
> That issue really stops my Dash application development

In the meantime, can you use the other method for Tabs, that is, rendering the children through a callback? Method 1 here: https://dash.plot.ly/dash-core-components/tabs

**Comment by valentijnnieman** [comment]
@radekwlsk #267 is merged and published as version 0.27.2!
